DuckDuckGo/AppIconSettingsViewController.swift

```diff
@@ -69,7 +69,7 @@ class AppIconDataSource: NSObject, UICollectionViewDataSource {
         cell.decorate(with: ThemeManager.shared.currentTheme)
 
         let appIcon = appIcons[indexPath.row]
-        cell.imageView.image = appIcon.mediumImage
+        cell.appIcon = appIcon
```

Comment:
Rather than passing in the image directly, pass in the `appIcon` so we can access its `name` to update the accessibility label.